apps/hexagon_matmul/Makefile

```diff
@@ -0,0 +1,55 @@
+include ../support/Makefile.inc
```

Comment:
It might literally just be HelloHexagon and this one, though I thought there were more. I copied from there in order to get Android support.

Maybe I'm thinking of simd_op_check (but that one probably needs it's own custom setup anyways). 

Either way... I'd like to make this fix in a separate PR.